user/abstractions/passphrase.md

```diff
@@ -41,6 +41,9 @@ Set a user's **passphrase**.
 * `passphrase`<br>
   The **passphrase** to set for the specified user or `False` to delete the current value.
 
+* `expired`<br>
+  A bool representing whether or not the **passphrase** should be set as **expired** and require changing on next logon.
```

Comment:
```markdown
* `expired`<br>
  A boolean toggle that is used to determine whether or not the user's **passphrase** should be set as **expired**,  meaning that the user will be required to change their **passphrase** on their next logon. If this argument is left unset, the user's **passphrase** will be set as **expired** by default.
```

Do the same for password as well.

I am pretty sure that the second sentence is correct, but if not, then it might be good to say that passwords/passphrase set this way usually default to expired and then just briefly explain the edge case.

Reply:
With the current code, the explanation is arguably correct, but I don't love it because it implies that if you don't supply `expired`, pyracf basically sets it to True which is not how the code is written (anymore) I changed it to the following.


```
* `expired`<br>
  A boolean toggle that is used to determine whether or not the user's **passphrase** should be set as **expired**,  meaning that the user will be required to change their **passphrase** on next logon. Without this argument specified, RACF will maek the user's **passphrase** as **expired** by default.
```

Reply:
So, given that we are not sure what the default behavior would be for security managers and because we don't want to imply that pyRACF is setting the `expired` flag, we specifically say that RACF will set the password/passphrase as expired by default.

Given, that I agree with this change. My only suggestion is a couple changes relating to prose and typo, and then I think we are good with this PR.

```markdown
* `expired`<br>
  A boolean toggle that is used to determine whether or not the user's **passphrase** should be set as **expired**,  meaning that the user will be required to change their **passphrase** on next logon. If this argument is not provided, RACF will set the user's **passphrase** as **expired** by default.
```

Reply:
Makes sense. This has been changed in both files now.